static.go

```diff
@@ -0,0 +1,7 @@
+package fakename
+
+//go:generate go run ./cmd/go-france generate
+
+func DefaultSet(opts ...NamesetOption) (fn, ln Nameset) {
```

Comment:
may be add some `TestDefaultSet` and update README.md#Use

Reply:
[13c5368](https://github.com/malikbenkirane/go-france/pull/2/commits/13c53686485dbdf9741bed8a07cfc20356722e4e)